logger/logger_test.go

```diff
@@ -51,7 +51,7 @@ func TestInitLogger(t *testing.T) {
 func TestInitLoggerFile(t *testing.T) {
 	dir := t.TempDir()
 	path := dir + "/test.log"
-	InitLogger(path, DebugLevel.String())
+	InitLogger(path, DebugLevel.String(), "test_instance")
```

Comment:
Currently we only test whether the logger is correctly initialized.
Consider adding a test for the prefix as well, e.g. 
```	// Check if the output starts with the instance_id
	if !strings.HasPrefix(string(data), "[test_instance]") {
		t.Errorf("Expected the output to start with '[test_instance]', but got: %s", string(data))
	}
```

Reply:
Thanks for the feedback and for testing the code. Will make the change!